server/fxa-mock.js

```diff
@@ -0,0 +1,62 @@
+const config = require('./config');
+
+const KEY_SCOPE = config.fxa_key_scope;
+let fxaConfig = null;
+let lastConfigRefresh = 0;
+
+const correct_token = 'correct_token.correct_token.correct_token';
+
+async function getFxaConfig() {
+  if (fxaConfig && Date.now() - lastConfigRefresh < 1000 * 60 * 5) {
+    return fxaConfig;
+  }
+  try {
+    fxaConfig = {
+      authorization_endpoint: 'https://accounts_enote/authorization',
+      introspection_endpoint: 'https://api-accounts_enote/v1/introspect',
+      issuer: 'https://accounts_enote',
+      jwks_uri: 'https://api-accounts_enote/v1/jwks',
+      revocation_endpoint: 'https://api-accounts_enote/v1/destroy',
+      token_endpoint: 'https://api-accounts_enote/v1/token',
+      userinfo_endpoint: 'https://profiles_enote/v1/profile',
+      claims_supported: ['aud', 'exp', 'iat', 'iss', 'sub'],
+      id_token_signing_alg_values_supported: ['RS256'],
+      response_types_supported: ['code', 'token'],
+      scopes_supported: ['openid', 'profile', 'email'],
+      subject_types_supported: ['public'],
+      token_endpoint_auth_methods_supported: ['client_secret_post']
+    };
+    fxaConfig.key_scope = KEY_SCOPE;
+    lastConfigRefresh = Date.now();
+  } catch (e) {
+    // continue with previous fxaConfig
+  }
+  return fxaConfig;
+}
+
+module.exports = {
+  token: correct_token,
+  getFxaConfig,
+  verify: async function(token) {
+    if (!token) {
+      return null;
+    }
+
+    if (token == correct_token) {
+      return {
```

Comment:
nit: for this we can use fixtures

Reply:
As per my understanding, the fixtures are used to make the code clearer (for exmple, hiding the auxilary code). 

Not sure if it make the code clearer here, as we can see exactly what's returned.

WDYT?

Reply:
Yep, this comment doesn't affect on the review decision overall. Just an observation